Tropos/Controllers/TRSettingsController.m

```diff
@@ -2,38 +2,32 @@
 #import "NSBundle+TRBundleInfo.h"
 
 NSString *const TRSettingsDidChangeNotification = @"TRSettingsDidChangeNotification";
-static NSString *const TRSettingsUnitSystemKey = @"TRUnitSystem";
-static NSString *const TRSettingsLastVersionKey = @"TRLastVersion";
+NSString *const TRSettingsUnitSystemKey = @"TRUnitSystem";
+NSString *const TRSettingsLastVersionKey = @"TRLastVersion";
 
 @interface TRSettingsController ()
 
-@property (nonatomic) RACSignal *userDefaultsChanged;
+@property (nonatomic) NSLocale *locale;
 
 @end
 
 @implementation TRSettingsController
 
 #pragma mark - Initializers
 
-- (instancetype)init
+- (instancetype)initWithLocale:(NSLocale *)locale
 {
     self = [super init];
     if (!self) return nil;
 
-    self.userDefaultsChanged = [[NSNotificationCenter defaultCenter] rac_addObserverForName:NSUserDefaultsDidChangeNotification object:nil];
+    self.locale = locale;
 
     return self;
 }
 
-#pragma mark - Properties
-
-- (RACSignal *)unitSystemChanged
+- (instancetype)init
 {
-    return [[[self.userDefaultsChanged filter:^BOOL(NSNotification *notification) {
-        return [notification.name isEqualToString:NSUserDefaultsDidChangeNotification];
-    }] map:^id(id value) {
-        return @([self unitSystem]);
-    }] startWith:@([self unitSystem])];
```

Comment:
This method has moved [here](https://github.com/thoughtbot/Tropos/pull/167/files#diff-50402482dc3839f4d529b6e1a18d494eR7).